server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreStatsIT.java

```diff
@@ -85,10 +129,28 @@ public void testStatsResponseAllShards() {
             .cluster()
             .prepareRemoteStoreStats(INDEX_NAME, null);
         RemoteStoreStatsResponse response = remoteStoreStatsRequestBuilder.get();
-        assertTrue(response.getSuccessfulShards() == 3);
-        assertTrue(response.getShards() != null && response.getShards().length == 3);
-        RemoteRefreshSegmentTracker.Stats stats = response.getShards()[0].getStats();
-        assertResponseStats(stats);
+        assertEquals(3, response.getSuccessfulShards());
+        assertTrue(response.getRemoteStoreStats() != null && response.getRemoteStoreStats().length == 3);
+        RemoteRefreshSegmentTracker.Stats stats = response.getRemoteStoreStats()[0].getStats();
+        validateUploadStats(stats);
+
+        // Step 3 - Enable replicas on the existing indices and ensure that download
+        // stats are being populated as well
+        changeReplicaCount(1);
+        remoteStoreStatsRequestBuilder = client(node).admin().cluster().prepareRemoteStoreStats(INDEX_NAME, null);
+        response = remoteStoreStatsRequestBuilder.get();
```

Comment:
nit - can we use `client(node).admin().cluster().prepareRemoteStoreStats(INDEX_NAME, shardId).get()`?

Reply:
If we are doing this, let's do it for other places as well.